soepy/exogenous_processes/children.py

```diff
@@ -31,12 +31,9 @@ def gen_prob_child_vector(model_spec):
 
     # Read data frame with information on probability to get a child
     # in every period
-    if "observed" in str(model_spec.kids_info_file_name).split("."):
-        exog_child_info_df = pd.read_pickle(str(model_spec.kids_info_file_name))
-    else:
-        exog_child_info_df = pd.read_pickle(
-            str(TEST_RESOURCES_DIR) + "/" + str(model_spec.kids_info_file_name)
-        )
+    exog_child_info_df = pd.read_pickle(
+        str(PACKAGE_DIR) + "/" + str(model_spec.kids_info_file_name)
```

Comment:
Why do you want to include the specification of the exogenous processes inside the package? These are application-specific. 

Why not simply:
`pd.read_pickle(str(model_spec.kids_info_file_name))`

Feel free to reach out later today if unclear.